internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java

```diff
@@ -416,6 +418,20 @@ public Optional<Version> getVersion(int versionNum) {
     return Optional.empty();
   }
 
+  /**
+   * Get backup version number, the largest online version number that is less than the current version number
+   */
+  public int getBackupVersionNumber() {
+    int currentVersion = getCurrentVersion();
+    getVersions().sort(Comparator.comparingInt(Version::getNumber).reversed());
+    for (Version v: getVersions()) {
```

Comment:
Nit: space for `Version v : getVersions()`

Reply:
Thanks. Will fix.

Reply:
Actually the format is changed by pre-commit checks. Even though I manually updated to `Version v : getVersions()`, after committing, it got changed back to `Version v: getVersions()`. I'll leave as is.